
#######
# ROS #
#######

ball_subscribe_topic: 'ball_relative'
ball_publish_topic: 'ball_relative_filtered'

#######
# UKF #
#######

filter_frequency: 10
